docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+services:
```

Comment:
let's rename this file to `compose.yaml` to align with rest of the company

Reply:
Done